app/javascript/components/concept-map/ConceptMap.tsx

```diff
@@ -57,7 +56,7 @@ export const ConceptMap = ({
     : new Set<string>()
 
   return (
-    <ExerciseStatusIndexProvider statusIndex={exerciseStatus}>
+    <>
```

Comment:
I had no idea what this did and I couldn't work out how to use it, so I removed it. If it's important then please revert this, and I'd appreciate you explaining it to me either way :)